app/api/helpers/permissions.py

```diff
@@ -157,10 +182,11 @@ def is_coorganizer(f):
     def decorated_function(*args, **kwargs):
         user = current_user
 
-        if user.is_staff:
-            return f(*args, **kwargs)
-        if 'event_id' in kwargs and user.has_event_access(kwargs['event_id']):
+        if user.is_staff or ('event_id' in kwargs and user.has_event_access(kwargs['event_id'])):
+            if 'event_identifier' in kwargs:
+                kwargs.pop('event_identifier', None)
```

Comment:
Why's this needed?

Reply:
Because then we will also have to pass an additional parameter of `event_identifier` in the function after using this decorator which is not used anywhere.

Reply:
But this can break a function where they actually are passing event_identifier

Reply:
@iamareebjamal Already considered this point. Check the implementation of the older `is_coorganizer` function, it always used to check using `event_id` only.

Reply:
That's not what I'm talking about. Consider a function where event_identifier is being received as a parameter and this decorator is applied there

Reply:
@iamareebjamal Considered this as well. It isn't the case anywhere right now.

Reply:
Right now isn't good enough, try to create a method with this decorator and see if it fails

Reply:
@iamareebjamal It will fail in case if the function wants event_identifier.